docs/build/program/apis/data-client.md

```diff
@@ -29,12 +29,13 @@ Data client API methods are only available in the Python SDK.
 
 ## Establish a connection
 
-To use the Viam data client API, you first need to instantiate a [`ViamClient`](https://python.viam.dev/autoapi/viam/app/viam_client/index.html#viam.app.viam_client.ViamClient) and then instantiate an [`DataClient`](https://python.viam.dev/autoapi/viam/app/data_client/index.html#viam.app.data_client.DataClient).
+To use the Viam data client API, you first need to instantiate a [`ViamClient`](https://python.viam.dev/autoapi/viam/app/viam_client/index.html#viam.app.viam_client.ViamClient) and then instantiate a [`DataClient`](https://python.viam.dev/autoapi/viam/app/data_client/index.html#viam.app.data_client.DataClient).
 See the following example for reference.
 
-<!-- After sveltekit migration we should also be able to get a key from the UI-->
+To get an API key and API key ID to authenticate, you have two options:
 
-Use the Viam CLI [to generate an API key to authenticate](/fleet/cli/#authenticate).
+- [Get an API key](/fleet/rbac/#add-an-api-key) from the [Viam app](https://app.viam.com), or
+- use the Viam CLI [to generate an API key](/fleet/cli/#authenticate).
```

Comment:
Nit: Ideally, let's use link title text that is itself a complete task (noun + verb). Especially helpful to differentiate between two similar options placed next to each other. What do you think of something like:

```suggestion
- [Create an API key using the Viam app](/fleet/rbac/#add-an-api-key)
- [Create an API key using the Viam CLI](/fleet/cli/#create-an-organization-api-key)
```

Since the first link directs us to the Viam app as part of _its_ guidance, I think having the Viam app link here might confuse, since there's no context to where in the App to go, what to click, like there is in the linked `rbac` resource. Also, one link per conceptual task option helps to keep choices straightforward.

Reply:
Great points. Applying your suggestion ty.